cadquery/occ_impl/assembly.py

```diff
@@ -50,9 +54,18 @@ def __init__(self, r: float, g: float, b: float, a: float = 0):
         """
         ...
 
+    @overload
```

Comment:
Is this still needed?

Reply:
It's not really needed (though will require a small change to the tests). 

`cq.Color()` creates a default color.  It is currently used in one of the tests where one object is assigned red and another object has an unspecified color. 

I thought it *might* be useful to create a default color.  It would allow you to do something like:

```python
mycolor = cq.Color()
mycolor.wrapped.SetAlpha(0.9)
```

Perhaps this is not very useful outside of the tests and so the Color change can be reverted.

Do you prefer to revert the `Color` change?  I'm fine with that and can update the tests tomorrow else please go ahead and revert and update the tests.

I'll add a separate comment on the test changes.

Reply:
OK, clear. Let's leave it in.